### PR TITLE
bootstrap added + css refactor

### PR DESCRIPTION
**Pull request by zbigniewkuminski**

### CODE REVIEW

File: src/components/AddPlayerToScoreboard/AddPlayerToScoreboard.js [hunk 1]
```diff
@@ -71,55 +71,62 @@ class AddPlayerToScoreboard extends React.Component {
...
+                {this.props.languageVersion?.scoreboardDescription.toUpperCase()}
+              </h2>
+            </div>
+            <div>
+              <button
+                className="button pull-right"
+                onClick={() => {
+                  this.closeScoreboard();
+                }}
+              >X</button>
```
Comment (saurongrajewo10) [inline src/components/AddPlayerToScoreboard/AddPlayerToScoreboard.js:96]: X button is not aligned to right side of the modal. Also in lower resolution X is displayed below header. Also highlight for save and x buttons should be implemented.

File: src/components/MainPage/MainPage.js [hunk 2]
```diff
@@ -8,13 +8,14 @@ class MainPage extends React.Component {
...
-    return (<div className={'welcome-screen ' + (this.state.enterAnimation ? "enter-game-animation " : "open-animation ")}>
-      <div className="title enter-game-animation-top">Hangman 2077</div>
-      <img src={require('./../../assets/hangman_home_page.gif')} className="hangman-gif" alt="test"></img>
-      <div >
-        <button className="button"
+    return (
+    <div className={'welcome-screen ' + (this.state.enterAnimation ? "enter-game-animation " : "open-animation ")}>
+      <div className="title enter-game-animation-top row justify-content-center">Hangman 2077</div>
+      <img src={require('./../../assets/hangman_home_page.gif')} className="hangman-gif row justify-content-center" alt="test"></img>
+      <div className="row justify-content-center">
```
Comment (saurongrajewo10) [inline src/components/MainPage/MainPage.js:15]: I would like to add a question, how does 'row' class work when it is not mentioned in MainPage.css? Also How 3 classNames are merged?

**Review by saurongrajewo10 — CHANGES_REQUESTED** [review]
Need work

**Review by saurongrajewo10 — APPROVED** [review]

**Comment by zbigniewkuminski** [comment]
Merged manually

**Pull request closed without merging**